### PR TITLE
Yet more pending transcations fixes...

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
- Open etherscan link in a new tab.
- Better detect "out of gas" messages.
- Bug that prevented un-minimizing notifications.

*Still left is to track down the bug that causes staking to not fire notifications on kovan.*

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
[Add an EtherScan link to each pending tx in notifications.](https://trello.com/c/wC7sxrmk/301-add-an-etherscan-link-to-each-pending-tx-in-notifications)

**Comment by dev-matan-tsuberi** [comment]
[Show tx Errors to the user.](https://trello.com/c/KSGzSFuG/238-show-tx-errors-to-the-user)

**Comment by dev-matan-tsuberi** [comment]
Fixed the mentioned problem by replacing all `Util.fromWei(..).toNumber()` with `Number(Util.fromWei(..))`.

`Util.fromWei(..).toNumber()` only works when `fromWei` returns a `BigNumber`, `Number(Util.fromWei(..))` should work even when it is a `string` or `number`.

@tibetsprague Please re-approve.

File: src/actions/arcActions.ts [hunk 1]
```diff
@@ -116,20 +116,20 @@ export async function getDAOData(avatarAddress: string, getDetails: boolean = fa
   const daoData: IDaoState = {
     avatarAddress,
     controllerAddress: "",
-    ethCount: Util.fromWei(await getBalance(avatarAddress)).toNumber(),
-    genCount: Util.fromWei((await votingMachineInstance.getTokenBalances({avatarAddress})).stakingTokenBalance).toNumber(),
+    ethCount: Number(Util.fromWei(await getBalance(avatarAddress))),
```
Comment (tibetsprague) [inline src/actions/arcActions.ts]: really still seems to me like we should just put the case in fromWei for now...

Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: https://github.com/daostack/alchemy/pull/121/commits/45e2bce7b100057a537367d55738a3bd8c19f205

**Pull request merged**